CONTRIBUTING.md

```diff
@@ -1,78 +1,116 @@
-# Contributing Guidelines
-If you're interested in contributing to this project, here are a few ways to do so:
+# Contribution Guidelines
 
-**Note:** If these contributing guidelines are not followed we will close the issue or PR, please
-read these instructions carefully.
+**Note:** If these contribution guidelines are not followed your issue or PR might be closed, so
+please read these instructions carefully.
 
-## Bug fixes
+## Contribution types
+
+### Bug fixes
  - If you find a bug, please first report it using
- [Github issues](https://github.com/flame-engine/flame/issues/new).
+   [Github issues](https://github.com/flame-engine/flame/issues/new).
```

Comment:
I would define the URL at the end of the page (similar to how it's done in the pull-request-template) to make line formatting easier.